CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.8.1 (2020-02-20)
+
+- Removed boolean as a valid type in the schema the `help` property.. [#41](https://github.com/blackbaud/skyux-config/pull/41)
```

Comment:
Got a double period at the end. Maybe, "Fixed the `help` property in the schema to only accept values of type `object`."?

Reply:
Incorporated feedback.